### PR TITLE
Acceptance Tests Structure and 1st test related CLI

### PR DESCRIPTION
**Pull request by davitol**
Acceptance Tests Structure Test for the 1st item https://github.com/owncloud/configreport/issues/66#issue-356997840
- [x] Run the occ command and check that the return code is 0

**How has it been tested**

Running test in local and passed.

```sudo -u www-data ./run.sh --config ../../apps/configreport/tests/acceptance/config/behat.yml --suite apiConfigReport```

```
  Scenario: admins generates a configreport                                 # /var/www/owncloud/apps/configreport/tests/acceptance/features/apiConfigReport/occConfigReportReturnCode.feature:7
[Thu Oct  4 05:07:39 2018] 127.0.0.1:56588 [200]: /ocs/v2.php/apps/testing/api/v1/occ
[Thu Oct  4 05:07:41 2018] 127.0.0.1:56594 [200]: /ocs/v2.php/apps/testing/api/v1/opcache
    Given the administrator has invoked occ command "configreport:generate" # FeatureContext::invokingTheCommand()
    Then the command should have been successful                            # FeatureContext::theCommandShouldHaveBeenSuccessful()
[Thu Oct  4 05:07:46 2018] 127.0.0.1:56598 [200]: /ocs/v1.php/apps/testing/api/v1/apps
[Thu Oct  4 05:07:47 2018] 127.0.0.1:41260 [200]: /ocs/v1.php/apps/testing/api/v1/apps
[Thu Oct  4 05:07:57 2018] 127.0.0.1:56606 [200]: /ocs/v2.php/cloud/apps?filter=disabled
[Thu Oct  4 05:08:06 2018] 127.0.0.1:56610 [200]: /ocs/v2.php/cloud/apps?filter=enabled

1 scenario (1 passed)
2 steps (2 passed)
1m0.91s (17.76Mb)
```

**Definition of Done**

- [x] acceptance test infrastructure
- [x] ``make test-acceptance-api`` Makefile target
- [x] first few test step definitions and feature file with scenarios that run
- [x] make the scenarios pass

### CODE REVIEW

File: tests/acceptance/features/apiConfigReport/occConfigReportReturnCode.feature [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@api
+Feature: generate a configreport via occ
+  As an administrator
+  I want to generate a configreport
```
Comment (PVince81) [inline tests/acceptance/features/apiConfigReport/occConfigReportReturnCode.feature]: why would I want one ? maybe add "to file a support / help request".
I don't think the main purpose is to check the server report but more to be able to get help on central / github / support

File: tests/acceptance/features/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (phil-davis) [inline tests/acceptance/features/.gitignore]: @davitol why did you need this?

File: tests/acceptance/features/bootstrap/ConfigReportContext.php [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+use Behat\Behat\Context\Context;
+use Behat\Behat\Hook\Scope\AfterScenarioScope;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use TestHelpers\AppConfigHelper;
+use TestHelpers\SetupHelper;
+
+require_once 'bootstrap.php';
+
+/**
+ * Context for password policy specific steps
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/ConfigReportContext.php]: s/password policy/config report/

For now, the context is empty, but we need an app-specific context to make Behat happy, so that is fine.
